contracts/Claims.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity ^0.7.5;
+
+// @notice Used to create Radicle Link identity claims
+contract Claims {
+    /// @notice Emitted on every Radicle Link identity claim
+    /// @param addr The account address which made the claim
+    event Claimed(address indexed addr);
+
+    /// @notice Creates a new claim of a Radicle Link identity.
+    /// Every new claim invalidates previous ones made with the same account.
+    /// The claims have no expiration date and don't need to be renewed.
+    /// @param version The version of `root` encoding, currently supported values:
+    /// - `1` - `root` is a SHA-1 root hash. The excess high bytes are zeros, e.g. for hash
+    /// `fb3102b74d7254eed7f18a31a3ba1ea946bb1a99` the passed `root` is
+    /// `000000000000000000000000fb3102b74d7254eed7f18a31a3ba1ea946bb1a99`
+    /// - `2` - `root` is a SHA-256 root hash
+    /// @param root The claimed Radicle Link identity root.
+    /// To revoke a claim without creating a new one, pass root `0`,
```

Comment:
Can't you pass `uint256(0)` to revoke the claim? Since there's only one valid claim per address?

Reply:
Yes, isn't that exactly what the documentation tells you to do to revoke the claim?

Reply:
:confused: I swear I read "Pass 0 as the version"